README.md

```diff
@@ -1,2 +1,127 @@
 # devices
 Servo-specific APIs to access various devices
+
+----------
+
+###How to use
+
+####Without the *bluetooth-test* feature:
+
+Each enum (`BluetoothAdapter`, `BluetoothDevice`, etc. ) will contain only one variant for each targeted platform. There are two supported platforms (android, linux), on other platforms we fall back to a default (empty) implementation. See the following `BluetoothAdapter` example:
```

Comment:
Remove the space after `etc.`
`android, linux` -> `Android, Linux`
`empty` -> `dummy` maybe